2004-08-03  Jeff Johnston  <jjohnstn@redhat.com>
   
        * gdb.arch/ia64-libunwind.exp: Fix test string to match
	current code base.

--- gdb+dejagnu-20040607/gdb/testsuite/gdb.arch/ia64-libunwind.exp.fix	Tue Aug  3 18:29:22 2004
+++ gdb+dejagnu-20040607/gdb/testsuite/gdb.arch/ia64-libunwind.exp	Tue Aug  3 18:41:01 2004
@@ -51,5 +51,5 @@ gdb_reinitialize_dir $srcdir/$subdir
 gdb_load ${binfile}
 send_gdb "set debug arch 1\n"
 send_gdb "break main\n"
-gdb_test "run" ".*acquire_unwind_info.*"
+gdb_test "run" ".*ia64_find_proc_info_x.*"
  
